### PR TITLE
feat: added styles file

### PR DESCRIPTION
**Pull request by RosaIsela31**
Changes: added styles file

### CODE REVIEW

**Review by luisduenas — APPROVED** [review]
LGTM

File: packages/gatsby-theme/src/gatsby-plugin-theme-ui/index.js [hunk 1]
```diff
@@ -1,4 +1,106 @@
...
+  intro: {
+    textTransform: "uppercase",
+    mb: ["xsmall", null, "small"],
+    mt: "xsmall",
+    fontWeight: "normal",
+    lineHeight: "normal",
+  },
+}
+const bShadows = {
+  default: "0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06)",
```
Comment (dinartecc) [inline packages/gatsby-theme/src/gatsby-plugin-theme-ui/index.js:48]: There is another shadow declaration that is currently being used in the Button component. We should add it.
https://github.com/octahedroid/drupal-gatsby-theme/blob/ad1ca942d8e864c2245f1947ad152d7b3b4e53c4/packages/gatsby-theme/src/components/Button/index.js#L29

Comment (RosaIsela31) [reply packages/gatsby-theme/src/gatsby-plugin-theme-ui/index.js:48]: Perfect! I'm going to add it. Thanks!

**Comment by omero** [comment]
:fire: Suggestion applied LGTM

**Pull request merged**